BeeJet.Storage/Databases/ButtonContextDb.cs

```diff
@@ -0,0 +1,46 @@
+﻿using BeeJet.Storage.Entities;
+using BeeJet.Storage.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BeeJet.Storage.Databases
+{
+    internal class ButtonContextDb : BaseDb<IButtonContext, ButtonContext>, IButtonContextDb
+    {
+        public ButtonContextDb(ILiteDatabase database)
+            : base(database)
+        {
+        }
+
+        protected override void EnsureIndexes()
+        {
+            Collection.EnsureIndex(b => b.Id);
+            Collection.EnsureIndex(b => b.MessageId);
+        }
+
+        public void CreateNewButtonContext(ulong messageId, string customIdButton, object context)
```

Comment:
Zou dit niet ook string moeten zijn dan? Als LiteDB inderdaad geen ulong ondersteunt, zou ik ze denk ik hier ook vermijden.

Reply:
Hij support wel ulong als het type echt ulong is. Maar als je type object hebt en er een ulong in gooit kan hij het niet resolven en wordt het een int64.

Reply:
Ahhh dat is gek. Ik las dat LiteDB überhaupt geen ulong ondersteunde vanwege de BSON specificatie :) Dan is het verder wel goed zo